lib/appsignal/config.ex

```diff
@@ -232,6 +257,24 @@ defmodule Appsignal.Config do
     |> Enum.into(%{})
   end
 
+  def single_line_suggested_request_headers do
+    @suggested_request_headers
+    |> List.flatten
+    |> Enum.join(",")
+  end
+
+  def multiline_suggested_request_headers do
+    Enum.map_join(@suggested_request_headers, "\n", fn(row) ->
+      "    #{Enum.join(row, " ")}"
+    end)
+  end
+
+  defp test? do
+    Mix.env
```

Comment:
@jeffkreeftmeijer It seems you've added a runtime dependency to Mix here, this breaks our distillery based releases which don't have Mix at runtime...  Just upgraded from 1.5.0 to 1.6.0 and our app no longer boots.

Reply:
Hi @doughsay , Sorry about that. Please downgrade to 1.5.0 until we've released the fix in PR https://github.com/appsignal/appsignal-elixir/pull/346 , which will hopefully be tomorrow!

Reply:
thanks! ;)

Reply:
Great job on fixing this so fast 🎉 

We have got the same issue as @doughsay with our distillery based deploys, so subscribe me to the next release notification :)

Reply:
@lucasmazza @doughsay We've released the 1.6.1 release yesterday that reverses this change. Please upgrade and let us know if you're running into any other issues.